web_widget_text_markdown/static/src/lib/bootstrap-markdown.js

```diff
@@ -0,0 +1,1575 @@
+/* ===================================================
```

Comment:
Isnot there a possibility not to embed the external library in the module and use better an external dependency?

Reply:
You mean use a CDN? Odoo can work on 'offline' environments (doesn't need an internet connection to run).
If you have security concerns can use tools like 'sha1sum' to check files integrity.